f3/api.go

```diff
@@ -4,7 +4,7 @@ package f3
 type ChainReceiver interface {
 	// Receives a chain appropriate for use as initial proposals for a Granite instance.
 	// The chain's base is assumed to be an appropriate base for the instance.
-	ReceiveCanonicalChain(chain ECChain, power PowerTable, beacon []byte)
+	ReceiveCanonicalChain(chain ECChain, power PowerTable, beacon []byte) error
 
 	// Receives a new EC chain, and notifies the current instance if it extends its current acceptable chain.
 	// This modifies the set of valid values for the current instance.
```

Comment:
I suggest adding an error return here too, even if currently it always succeeds. For consistency and preparation for future changes that may need an error.

Reply:
Makes sense. Done.